sdk/digitaltwins/azure-digitaltwins-core/src/main/resources/azure-digital-twins.properties

```diff
@@ -0,0 +1,2 @@
+name=${project.artifactId}
+version=${project.version}
```

Comment:
"azure-digitaltwins-core" and "1.0.0-beta.1" as defined in the pom file right now